### PR TITLE
Make some braintree-web/apple-pay methods static according to apple p…

### PR DESCRIPTION
**Pull request by ONE001**
Adding missing static type definition for ApplePaySession which determines support for API and Payments according to Apple pay documentation https://developer.apple.com/documentation/apple_pay_on_the_web/applepaysession

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Review by jbuckner — APPROVED** [review]
Looks good! Thanks for the update!

**Comment by ONE001** [comment]
When do you think it's going to be merged?

**Comment by jbuckner** [comment]
Hmm usually the `typescript-bot` will post a message after it's been approved to merge it. (I don't have merge access). I wonder if it's because the tests haven't changed. There's a line up there that says:
`Only a DT maintainer can approve changes without tests`

**Comment by ONE001** [comment]
Hey @jbuckner, I just added some tests, could you please take a look?

**Review by jbuckner — APPROVED** [review]
Looks good!

**Comment by ONE001** [comment]
Ready to merge

**Pull request merged**
